app/assets/v2/js/shared.js

```diff
@@ -977,6 +977,20 @@ function getNetwork(id) {
   return networks[id] || 'custom network';
 }
 
+function resolveENS(ens_name) {
+  let test_Url = 'http://localhost:8000/resolve_ens/';
+  let prod_Url = 'https://gitcoin.co/resolve_ens/';
+
+  var request = new XMLHttpRequest();
```

Comment:
^ let /const 
Do we wanna use `fetchData` instead of `XMLHttpRequest`?

Reply:
for sure we should be using fetch instead.